src/components/Questionnaire/ManageResponseTemplatesSheet.tsx

```
@@ -0,0 +1,1075 @@
+                  <ClipboardListIcon className="size-2.5" />
+                  {activityDefinitionCount}
+                </Badge>
+              )}
+            </div>
+          </div>
+
+          <div
+            className="flex items-center gap-1 shrink-0"
+            onClick={(e) => e.stopPropagation()}
+          >
+            {isApplying && (
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)